ES(Spanish)/README.md

```diff
@@ -0,0 +1,361 @@
+# :wave:  Bienvenidos al Código para la Comunidad
+<p align="center"><img src="https://user-images.githubusercontent.com/75534912/194343020-6cbd2485-c2ea-4779-87e2-9708238398e7.png"></p>
+# La entrada del código abierto | Los mejores recursos de ls programación
+## Nuestros objetivos:rocket:
+Los objetivos de la comunidad no son solo proveer los mejores recursos, sino también <b>motivatar y animar los principantes que no han contruibido nunca al código abierto, y ayudarlos a empezar</b>proyectos fácilmente y con apoyo total.</p>
+
+## Comparte tu contribución
+
+**No te olvides de compartir tu contribuciön en redes sociales. Eso motivará otras personas para que comiencen :rocket: su aventura del código abierto.**
+
+### **Únete la comunidad:people_holding_hands: y connecta conmigo :handshake:**
+
+<div align="center">
+<a href="https://twitter.com/codeforcomm"> <img src="https://img.shields.io/badge/Twitter-%231DA1F2CFC.svg?style=for-the-badge&logo=Twitter&logoColor=white"/><a>
+<a href="https://discord.gg/AfYcurdCd3"><img src="https://img.shields.io/badge/%3CCode For Community%3E-%237289DA.svg?style=for-the-badge&logo=discord&logoColor=white"/></a>
+<a href="https://twitter.com/iashishkhangwal"> <img src="https://img.shields.io/badge/Twitter-%231DA1F2.svg?style=for-the-badge&logo=Twitter&logoColor=white"/><a>
+<a href="https://www.linkedin.com/in/ashish-khanagwal-890326213/"><img src="https://img.shields.io/badge/linkedin-%230077B5.svg?style=for-the-badge&logo=linkedin&logoColor=white"/></a>
+</div>
+
+## La guía para contribuir :point_down:
+  
+Para añadir o contribuir lee  [Contribution-guide](https://github.com/Ashish-khanagwal/Open-source-practice-and-resources/blob/main/CONTRIBUTING.md) 
+  
+**Añade recursos en orden alfabético y sigue las directrices de la contribución**
+  
+---  
+  
+**Usa estos emojis para organizar los recursos**
+  
+:file_folder: Documentación
+
+:speaker: Podcast
+
+:books: Blogs/Listas de tutoriales
+
+:hammer_and_wrench: Instrumentos/Softwares/Hojas de rutas
+
+:video_camera: Recursos de vídeo (Youtube, cursos, etc...)
+
+:bulb: Otras/Sugerencias
+
+---  
+  
+## Índice
```

Comment:
Please remove all the resources from the line `44`, as it is not needed in this particular file.